mails/themes/modern/modules/followup/followup_2.html.twig

```diff
@@ -0,0 +1,228 @@
+{% extends '@MailThemes/modern/components/layout.html.twig' %}
+        
+{% block title %}{{ 'Follow up 2'|trans({}, 'Emails.Body', locale) }}{% endblock %}
```

Comment:
Is it a new string? 🤔 I can't find it on Crowdin.

Reply:
@jolelievre
Can you answer @Julievrz ?
I based my work on `master` branch. Was this correct ?

Reply:
@Julievrz 
This is not a new string. It is already there in https://github.com/PrestaShop/PrestaShop/blob/develop/mails/themes/modern/modules/ps_reminder/followup_2.html.twig#L3